database/cockroachdb/TUTORIAL.md

```diff
@@ -0,0 +1,142 @@
+# CockroachDB tutorial for beginners (insecure cluster)
+
+## Create/configure database
+
+First, let's start a local cluster - follow step 1. and 2. from [the docs](https://www.cockroachlabs.com/docs/stable/start-a-local-cluster.html#step-1-start-the-first-node).
+
+Once you have it, create a database. Here I am going to create a database called `example`.
+Our user here is `cockroach`. We are not going to use a password, since it's not supported for insecure cluster.
+```
+cockroach sql --insecure --host=localhost:26257
+```
+```
+CREATE DATABASE example;
+CREATE USER IF NOT EXISTS cockroach;
+GRANT ALL ON DATABASE example TO cockroach;
+```
+
+When using Migrate CLI we need to pass to database URL. Let's export it to a variable for convienience:
+```
+export COCKROACHDB_URL=cockroachdb://cockroach:@localhost:26257/example?sslmode=disable
```

Comment:
To prevent shell interpetation, use `export COCKROACHDB_URL='cockroachdb://cockroach:@localhost:26257/example?sslmode=disable'` (note the single quotes)

Reply:
Good catch, I will fix it